sdk/src/test/scala/com/horizen/validation/ConsensusValidatorOmmersTest.scala

```diff
@@ -2,26 +2,30 @@ package com.horizen.validation
 
 import com.horizen.SidechainHistory
 import com.horizen.block.{Ommer, SidechainBlock, SidechainBlockHeader}
-import com.horizen.consensus._
+import com.horizen.chain.SidechainBlockInfo
+import com.horizen.consensus.{ConsensusEpochNumber, _}
 import com.horizen.fixtures.{CompanionsFixture, SidechainBlockFixture, TransactionFixture}
 import com.horizen.params.{MainNetParams, NetworkParams}
-import org.junit.Assert.{assertEquals, assertArrayEquals, fail => jFail}
+import com.horizen.proof.VrfProof
+import com.horizen.vrf.VrfProofHash
+import org.junit.Assert.{assertArrayEquals, assertEquals, fail => jFail}
 import org.junit.Test
 import org.mockito.{ArgumentMatchers, Mockito}
 import org.scalatest.junit.JUnitSuite
 import org.scalatest.mockito.MockitoSugar
 import scorex.util.ModifierId
 
+import scala.collection.mutable.ListBuffer
 import scala.util.{Failure, Success, Try}
 
 class ConsensusValidatorOmmersTest
   extends JUnitSuite
     with MockitoSugar
   with CompanionsFixture
   with TransactionFixture
-  with SidechainBlockFixture{
+  with SidechainBlockFixture {
```

Comment:
do you have a test where you check nonce calculation equality of nonces done by a direct method (i.e. if epoch had been ended), and nonce calculation for ommers?